tools/otel-collector/otel-config.yaml

```diff
@@ -0,0 +1,74 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/dash0hq/otelbin/refs/heads/main/packages/otelbin/src/components/monaco-editor/schema.json
+
+# service.pipelines pull the configured receivers, processors, and exporters together into
+# pipelines that process data.
+#
+# receivers, processors, and exporters that are not used in pipelines are silently ignored.
+service:
+  extensions: [health_check, zpages]
+  pipelines:
+    logs:
+      receivers: [otlp]
+      processors: [memory_limiter, batch]
+      exporters: [debug, otlp/jaeger]
+    traces:
+      receivers: [otlp]
+      exporters: [debug, otlp/jaeger, spanmetrics]
+    metrics/spanmetrics:
+      receivers: [spanmetrics]
+      exporters: [debug, prometheus]
```

Comment:
this is the YAML config file for the local open-telemetry collector. A similar YAML exists in Argo for the live environments